pkg/pdsutils/lib/environment.go

```diff
@@ -0,0 +1,33 @@
+package lib
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	envControlPlaneURL        = "CONTROL_PLANE_URL"
```

Comment:
Instead pass these variables as -e CONTROL_PLANE_URL=<>.... then take it in deploy-ssh.sh script. 
Pass it pass flag to the code and set variable just follow the example of `TESTRAIL_MILESTONE`